core/standards.md

```diff
@@ -0,0 +1,80 @@
+# Core Smart Contracts
+
+## Definition of Core
+
+A core contract is one that:
+
+1. Any contract that is part of the [dss repo](https://github.com/makerdao/dss)
+2. Will have ward status on or other privileged access to any other core contract (i.e. `Vat.rely(<new contract address>)`)
+3. Will rely on the Pause Proxy as a permissioned actor. If the pause proxy has to directly interact with this contract then there is additional security and process work that needs to be done.
+4. Will be included in the chainlog as a canonical “Maker Contract”. Even if the contract does not have elevated permissions in the system, a failure of a contract in the chainlog could result in cost (bug bounties) or reputation damage for the protocol.
+5. If a contract is put forward to a business partner of the DAO, by DAO members, close consideration should be given to whether it would be perceived as core Maker brand by those partners.
+
+## Protocol Engineering Review Process for Core Contracts
+
+All Core contracts need to go through a risk adjusted review process.  Risk should be evaluated at the beginning of the project and continue to be re-evaluated through the engineering process.
+
+Our review process at a minimum requires two team member reviews and approvals.  However for higher risk contracts may also involve Red Team reviews and audits by external auditors.
+
+Core contracts should follow established coding patterns including repo and contract structures, variable naming patterns (yes Daiwanese is a standard for our team) and auth patterns.
```

Comment:
It goes without saying (at least it should) that we value security over gas optimisations and value costly simplicity over cheap complexity. As such, low level calls should be used minimally and gas optimisation tricks should be used where appropriate but not any more.

Just my 2c here

Reply:
Added some comments here: https://github.com/makerdao/pe-checklists/pull/2/commits/be221016ee1746864099252b6760156ff4e16211